config/dynamicconfig/development.yaml

```diff
@@ -13,6 +13,8 @@ history.useNewInitialFailoverVersion:
 history.enableStrongIdempotency:
 - value: true
   constraints: {}
+frontend.globalRatelimiterMode:
+- value: local
```

Comment:
ensures we integration-test with the newer behavior, and seems worth defaulting to in dev either way (no constant, idle request flow causing noise)